File: mycollab-web/src/main/java/com/mycollab/module/project/view/settings/ProjectMemberListView.java
package com.mycollab.module.project.view.settings;

import com.mycollab.module.project.domain.criteria.ProjectMemberSearchCriteria;
import com.mycollab.vaadin.mvp.PageView;

/**
 * @author MyCollab Ltd.
 * @since 1.0
 */
public interface ProjectMemberListView extends PageView {
    void setSearchCriteria(ProjectMemberSearchCriteria searchCriteria);
}

File: mycollab-mobile/src/main/java/com/mycollab/mobile/module/project/view/settings/ProjectMemberListView.kt
package com.mycollab.mobile.module.project.view.settings

import com.mycollab.mobile.ui.IListView
import com.mycollab.module.project.domain.SimpleProjectMember
import com.mycollab.module.project.domain.criteria.ProjectMemberSearchCriteria

/**
 * @author MyCollab Ltd.
 * @since 4.5.2
 */
interface ProjectMemberListView : IListView<ProjectMemberSearchCriteria, SimpleProjectMember>
